airflow/configuration.py

```diff
@@ -2166,8 +2163,8 @@ def initialize_auth_manager() -> BaseAuthManager:
     TEST_PLUGINS_FOLDER = os.path.join(AIRFLOW_HOME, "plugins")
 
 SECRET_KEY = b64encode(os.urandom(16)).decode("utf-8")
-JWT_SECRET_KEY = b64encode(os.urandom(16)).decode("utf-8")
 FERNET_KEY = ""  # Set only if needed when generating a new file
+JWT_SECRET_KEY = ""
```

Comment:
Maybe with this change I should have the JWTSigner create a default value if the config is empty and issue a warning, that would improve the upgrade path.

Reply:
I think that's a good idea indeed.

Reply:
I agree. Otherwise that could be frustrating for users

Reply:
Handled in b7fc679ac4